sql/python/sqlflow_submitter/db_test.py

```diff
@@ -39,20 +40,27 @@ def test_mysql(self):
             password = os.environ.get('SQLFLOW_TEST_DB_MYSQL_PASSWD') or "root"
             host = "127.0.0.1"
             port = "3306"
-            database = "iris"
-            conn = connect(driver, database, user=user, password=password, host=host, port=port)
```

Comment:
Python tests shouldn't assume the existence of `iris` database which is created by go test.

Reply:
Seems `test_generator` I added before need this update too. Would you mind updating that part please? Thanks

Reply:
@typhoonzero thanks for reminding. Will do!